homeassistant/components/conversation/__init__.py

```diff
@@ -120,7 +133,7 @@ async def websocket_set_onboarding(hass, connection, msg):
     if success:
         connection.send_result(msg["id"])
     else:
-        connection.send_error(msg["id"])
+        connection.send_error(msg["id"], "error", "Failed to set onboarding")
```

Comment:
`code` and `message` are compulsory, but were missing.

https://github.com/home-assistant/core/blob/6b256bab227346bdd6d0ae871855b70ebefbba01/homeassistant/components/websocket_api/connection.py#L58-L60